README.md

```diff
@@ -0,0 +1,21 @@
+# Cloudskiff
+Cloudskiff measures infrastucture as code coverage, and tracks cloud configuration drift.
+
+## Why ?
+
+Infrastructure as code is awesome, but there are too many moving parts: codebase, state file, actual cloud configuration. Things tend to drift.
+
+Drift can have multiple causes: from developers creating or updating infrastructure through the web console without telling anyone, to uncontrolled updates on the cloud provider side. Handling infrastructure drift vs the codebase can be challenging.
```

Comment:
WDYT? "...to uncontrolled updates on the cloud provider side, _or undetected changes by the IaC tool_."